src/database/migrations/1755331547402-InsertChiefTechnologist.ts

```diff
@@ -0,0 +1,35 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import {staffConfig} from "@/config/config";
+
+export class InsertChiefTechnologist1755331547402 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`
+            INSERT INTO lab_staffs ( 
+            first_name,
+            last_name,
+            email,
+            phone_number,
+            address,
+            password,
+            lab_staff_role,
+            is_active
+            )
+            VALUES (
+            '${staffConfig.chiefTechnologist.firstName}',
+            '${staffConfig.chiefTechnologist.lastName}',
+            '${staffConfig.chiefTechnologist.email}',
+            NULL,
+            NULL,
+            'temp1234',
+            'Chief Technologist',
```

Comment:
you've not done what I asked you to do. 
1. you should be loading this from your environment
2. why is it still plain string. whats now the point of bcrypt?